### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

**Comment by ronenkat** [comment]
@hunchback we should make sure our PRs have proper decriptions

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [![Build Status](https://travis-ci.com/IBM/the-mesh-for-data.svg?branch=master)](https://travis-ci.com/IBM/the-mesh-for-data "Travis")
```
Comment (roee88) [inline README.md:2]: I think that the empty line should be removed

Comment (hunchback) [reply README.md:2]: i agree but not sure how to edit from github WebUI (via my mobile)

Comment (roee88) [reply README.md]: Fixed

**Review by roee88 — APPROVED** [review]

**Pull request merged**